Index: xombrero-1.3.0/xombrero.h
===================================================================
--- xombrero-1.3.0.orig/xombrero.h
+++ xombrero-1.3.0/xombrero.h
@@ -660,7 +660,7 @@ int		command_mode(struct tab *, struct k
 #define XT_DS_ALLOW_INSECURE_CONTENT	(TRUE)
 #define XT_DS_ALLOW_INSECURE_SCRIPTS	(TRUE)
 #define XT_DS_WARN_CERT_CHANGES	(0)
-#define XT_DS_RESOURCE_DIR	("/usr/local/share/xombrero")
+#define XT_DS_RESOURCE_DIR	("/usr/share/xombrero")
 #define XT_DS_DO_NOT_TRACK	(0)
 #define XT_DS_PRELOAD_STRICT_TRANSPORT	(1)
 #define XT_DS_GNUTLS_PRIORITY_STRING	(NULL)
